packages/@angular/cli/utilities/stats.ts

```diff
@@ -41,8 +39,24 @@ export function statsToString(json: any, statsConfig: any) {
         .join('');
 
       return `chunk {${y(chunk.id)}} ${g(files)}${names}${size} ${initial}${flags}`;
-    }).join('\n')}
-    `);
+    });
+
+  const unchangedChunkNumber = json.chunks.length - changedChunksStats.length;
+
+  if (unchangedChunkNumber > 0) {
+    return rs(stripIndents`
+      Date: ${w(new Date().toISOString())} • Hash: ${w(json.hash)} • Time: ${w('' + json.time)}ms
```

Comment:
This output leads to two beep sounds on windows systems due to the two '•' characters, see https://blogs.msdn.microsoft.com/oldnewthing/20070104-00/?p=28513

Reply:
Let's use a fancy character they said. What could go wrong they said.

😞

Reply:
@TobiasSchulte does this happen on your system? I tried on mine (Windows 10, basic `cmd`) and I didn't get a beep.

Reply:
@filipesilva Yes, it happens on my machine and on the machine of a colleague. Both are Windows 7 systems, it happens in a normal cmd, but not for example in the vscode integrated terminal.

Reply:
Would you be interested in making a PR that changes `•` to `-`?

Reply:
@filipesilva Pull request is created

Reply:
@filipesilva  looks like a global bug in Win7 x64 atm (both Professional and Home Premium versions checked yet) - ```•``` bullet characters are causing aroung +-3 mins in recompiling process due to 'beep-loop' during any 'ng serve' operations + freezing cmd ;)

Reply:
@TobiasSchulte we have already made that change in https://github.com/angular/angular-cli/pull/9336